### PR TITLE
MNT Update how phpcs is called

### PR DESCRIPTION
**Pull request by emteknetnz**
Fixes https://travis-ci.com/github/silverstripe/silverstripe-reports/jobs/423658754

I copied the phpcs line from
https://github.com/silverstripe/silverstripe-travis-shared/blob/master/config/script/phpcs.yml

### CODE REVIEW

**Review by robbieaverill — APPROVED** [review]
Paths are configured in the shared config I assume?

**Review by dnsl48 — CHANGES_REQUESTED** [review]
phpcs paths should go to `phpcs.xml.dist` config.
here's an example - https://github.com/silverstripe/silverstripe-fulltextsearch/pull/291/files

**Review by dnsl48 — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,51 +1,9 @@
-language: php
+# https://docs.travis-ci.com/user/build-config-imports/
```
Comment (emteknetnz) [inline .travis.yml:1]: probably don't need this comment in the file

Comment (emteknetnz) [inline .travis.yml:1]: probably don't need this comment in the file

**Review by emteknetnz — COMMENTED** [review]
Exciting :-)

File: .travis.yml (hunk 1 above)
Comment (dnsl48) [reply .travis.yml]: done

File: .travis.yml [hunk 2]
```diff
@@ -1,51 +1,9 @@
...
-  - composer update --prefer-dist --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile $COMPOSER_ARG
-
-script:
-  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit tests; fi
-  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover=coverage.xml tests; fi
-  - if [[ $PHPCS_TEST ]]; then composer run-script lint; fi
-
-after_success:
-  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then bash <(curl -s https://codecov.io/bash) -f coverage.xml; fi
+    - COMPOSER_ROOT_VERSION="4.x-dev"
```
Comment (emteknetnz) [inline .travis.yml:8]: This should be 4.7.x-dev

Comment (dnsl48) [reply .travis.yml:8]: nice catch!

**Comment by dnsl48** [comment]
@emteknetnz we'll have to merge this one first - https://github.com/silverstripe/silverstripe-installer/pull/292

**Pull request merged**